docs/sources/datasources/loki.md

```diff
@@ -213,13 +213,14 @@ Check out the [Templating]({{< relref "../variables/" >}}) documentation for an
 ## Query variable
 
 Variable of the type _Query_ allows you to query Loki for a list labels or label values. The Loki data source plugin
-provides the following functions you can use in the `Query` input field.
-
-| Name                                       | Description                                                                            |
-| ------------------------------------------ | -------------------------------------------------------------------------------------- |
-| `label_names()`                            | Returns a list of label names.                                                         |
-| `label_values(label)`                      | Returns a list of label values for the `label`.                                        |
-| `label_values(log stream selector, label)` | Returns a list of label values for the `label` in the specified `log stream selector`. |
+provides a form to select the type of values that can be expected to be used for a given variable. This form has the following
+options to choose from:
```

Comment:
The second sentence ("The Loki data source...") was a little hard to parse. I tried to clarify the wording on this sentence. Is this still correct? 
```suggestion
provides a form to select the type of values that can be expected for a given variable.
The form has these options:
```

Reply:
Updated. Thanks!